src/components/portal/portal.js

```diff
@@ -53,15 +57,15 @@ const Portal = ({ children, className, id, onReposition }) => {
       }
       setPortalNode(node);
 
+      let mainNode = document.body;
       const rootDiv = document.getElementById("root");
 
-      // Storybook adds the hidden attribute to the root element when on the "Docs" pages
-      // Without this check the portal would also be hidden
-      if (rootDiv && !rootDiv.hasAttribute("hidden")) {
-        rootDiv.appendChild(node);
-      } else {
-        document.body.appendChild(node);
+      if (rootDiv && isChromatic()) {
```

Comment:
I don't want us to add the `chromatic` package as a dependency. Is there another way to achieve this?

I think at a minimum we should implement a context provider to abstract this away from our consumers, we can then wrap our stories in the context provider that provides the chromatic specific logic.

Reply:
good point, I have wrapped it in a context, and the value is set in storybook now